libraries/AP_HAL_Linux/Storage.h

```diff
@@ -2,7 +2,8 @@
 #define __AP_HAL_LINUX_STORAGE_H__
 
 #if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ERLEBOARD || CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_PXF
-#define LINUX_STORAGE_USE_FRAM 1
+#define LINUX_STORAGE_USE_FRAM 0
```

Comment:
A quick fix as a separate commit (on top of this one) has been added in another branch https://github.com/erlerobot/ardupilot/commit/b812b44e9ee543cae585085ea95dde0ee621d1b4.

I may have another look in the future but for now I'm most concerned about getting this existing PR into master.